sources/views/settings/tableConstructor.js

```diff
@@ -0,0 +1,60 @@
+import {JetView} from "webix-jet";
+
+export default class Table extends JetView {
+	constructor(app, name, data) {
+		super(app, name);
+		this._tableData = data;
+	}
+
+	config() {
+		const _ = this.app.getService("locale")._;
+		return {
+			rows: [
+				{
+					view: "datatable",
+					localId: "table",
+					editable: true,
+					select: true,
+					columns: [
+						{id: "Value", header: _("Value"), fillspace: true, editor: "text"},
+						{id: "Icon", header: _("Icon"), editor: "text", template: obj => `<span class='mdi mdi-${obj.Icon}'></span>`}
```

Comment:
Names of icons are not obvious for users.
Better use richselect/combo editor with a set of 5-7 icons that will be visible (icon+title) in options. Use the native webix [icons](https://snippet.webix.com/honyfqgt) or the [ones](http://materialdesignicons.com/cdn/2.7.94/) you use in the app.